gui/mocked_keycard_controller.py

```diff
@@ -71,3 +84,50 @@ def input_custom_keycard_details(self, details: str, index: int):
         self._scroll.vertical_scroll_to(QObject(name='', real_name=driver.objectMap.realName(fields[index])))
         driver.type(fields[index], details)
         time.sleep(1)
+
+    @allure.step('Choose not Status keycard from initial keycard state dropdown')
+    def choose_not_status_keycard(self):
+        if not self._keycard_state_button.is_visible:
+            self._scroll.vertical_scroll_to(self._keycard_state_button)
+        self._keycard_state_button.click()
+        self._not_status_keycard_item.click()
+        time.sleep(1)
+        return self
+
+    @allure.step('Choose empty keycard from initial keycard state dropdown')
+    def choose_empty_keycard(self):
+        if not self._keycard_state_button.is_visible:
+            self._scroll.vertical_scroll_to(self._keycard_state_button)
+        self._keycard_state_button.click()
+        self._empty_keycard_item.click()
+        time.sleep(1)
+        return self
+
+    @allure.step('Choose keycard with MAX pairing slots reached from initial keycard state dropdown')
+    def choose_max_slots_reached_keycard(self):
+        if not self._keycard_state_button.is_visible:
+            self._scroll.vertical_scroll_to(self._keycard_state_button)
+        self._keycard_state_button.click()
+        self._max_slots_reached_item.click()
+        time.sleep(1)
+        return self
+
+    @allure.step('Choose keycard with MAX pairing slots reached from initial keycard state dropdown')
+    def choose_max_slots_reached_keycard(self):
```

Comment:
Why do these methods have the same name?

Reply:
I don't know why these steps were duplicated, deleted one of them